### ISSUE TITLE
setting up cluster registry fails 

### ISSUE THREAD

**Issue by mmeliani** [kubernetes/cluster-registry#230]
get the following error while installing the cluster registry 
```
kubectl apply -f cluster-registry-crd.yaml 
The CustomResourceDefinition "clusters.clusterregistry.k8s.io" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[authInfo].properties[providers].items.properties[config].additionalProperties: Forbidden: additionalProperties cannot be set to false
```
where am I going wrong ?  

/sig multicluster

**Comment by perotinus** [comment]
@mmeliani This is an issue with the OpenAPI spec validation. I believe this has been fixed in Kubernetes HEAD and will be released in 1.11 (https://github.com/kubernetes/kubernetes/pull/62333). For now, you can delete that (as in #231) and it should work.

**Comment by perotinus** [comment]
@mmeliani #231 has merged, so I believe this should be fixed now.
